tools/test.py

```diff
@@ -1265,6 +1265,9 @@ def BuildOptions():
   result.add_option("--no-store-unexpected-output", 
       help="Deletes the temporary JS files from tests that fails",
       dest="store_unexpected_output", action="store_false")
+  result.add_option("-r", "--run",
+      help="Run every mth test starting from n (--run=n,m with n < m)",
```

Comment:
I think the word 'every' is more clear than before, but 'starting from' might be slightly confusing.
How about `Run every n-th test modulo m` ? The point is not to skip n tests, is to have m groups, and be able to specify which group to run.